### ISSUE TITLE
[BUG] Group manager: unintended expanding of subcategories 

### ISSUE THREAD

**Issue by Danny-dK** [UtrechtUniversity/yoda-ansible#409]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Difficult to really indicate the steps as it at times just happens random. If I expand 2 main categories and start expanding subcategories within 1 of the main categories, a subcategory from the other main category expands. Hope this screen record makes it a bit more clear where I click on multiple sub-categories in 2 main categories:
[group_manager_yoda_issue.webm](https://github.com/UtrechtUniversity/yoda/assets/45395070/4a774afb-8acc-428f-8ffb-4149ea8d200c)

### Expected Behavior

To not start expanding subcategories of the other expanded main category.

### Steps To Reproduce

- expand 2 main categories that are next to each other 
- expand a subcategory in one of the main categories, which triggers opening of another subcategory in the other main category 

### Environment

```markdown
- Yoda: Yoda release-1.9
- Operating System: Windows 10
- Browser:Firefox 124.0.2 (64-bit)
```

**Comment by stsnel** [comment]
Thank you for reporting this issue. We can reproduce it. Internal issue number is YDA-5667. We aim to include a fix in v1.9.3

**Comment by lwesterhof** [comment]
Fixed in https://github.com/UtrechtUniversity/yoda-portal/commit/d0362dbc3a64e819e48c6576e1688f76090c34d5
